Patterns/Swedish/Swedish-TimeZone.yaml

```diff
@@ -692,9 +684,7 @@ AbbrToMinMapping: !dictionary
    kost: 660
    lhdt: 660
    lhst: 630
-   #mart: 570
-   # Used mart entry is for Marquesas Islands (French Polynesia) in the Pacific Ocean
-   mart: -570
+   mart: -10000
```

Comment:
In such ambiguous cases, we should keep the possible values also in the YAML file as documentation of the ambiguity. In the future, these can be used to offer alternatives.